File: 2026-08/src/day_05/prms_120899/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(vector<int> array) {
    int mx = -1;
    int idx = -1;

    for (int i = 0; i < array.size(); i++) {
        if (array[i] > mx) {
            mx = array[i];
            idx = i;
        }
    }

    return {mx, idx};
}

File: 2026-08/src/day_05/prms_120899/Solution.java
package day_05.prms_120899;

class Solution {
    public int[] solution(int[] array) {
        int max = -1;
        int idx = -1;

        for (int i = 0; i < array.length; i++) {
            if (array[i] > max) {
                max = array[i];
                idx = i;
            }
        }

        return new int[]{max, idx};
    }
}
